megatron/neox_arguments/arguments.py

```diff
@@ -359,16 +367,20 @@ def consume_deepy_args(cls):
         ]
 
         # determine overwrite values
-        overwrite_values = dict()
-        for k, v in vars(args_parsed).items():
-            if k == "autotuning" and v is not None:
-                overwrite_values["autotuning_run"] = v
-            elif k not in ["conf_dir", "conf_file"] and v is not None:
-                overwrite_values[k] = v
+        possible_args = inspect.getfullargspec(cls)[0]
+        full_overwrite_values = dict()
+        for k, v in args_parsed.items():
+            if (
+                k not in ["conf_dir", "conf_file"]
```

Comment:
@dashstander -- Will removing the `overwrite_values["autotuning_run"]` break your autotuning code here?

Reply:
Yes, that block is there for a reason. Did it absolutely need to go?